parser.go

```diff
@@ -4462,6 +4468,306 @@ func (p *Parser) tryParseTablePrivilegeColumns() ([]*ast.Ident, token.Pos) {
 	return columns, rparen
 }
 
+// begin CREATE PROPERTY GRAPH
+
+func (p *Parser) parseCreatePropertyGraph(pos token.Pos, orReplace bool) *ast.CreatePropertyGraph {
+	p.expectKeywordLike("PROPERTY")
+	p.expectKeywordLike("GRAPH")
+
+	ifNotExists := p.parseIfNotExists()
+	name := p.parseIdent()
+	content := p.parsePropertyGraphContent()
+
+	return &ast.CreatePropertyGraph{
+		Create:      pos,
+		OrReplace:   orReplace,
+		IfNotExists: ifNotExists,
+		Name:        name,
+		Content:     content,
+	}
+}
+
+func (p *Parser) parsePropertyGraphContent() *ast.PropertyGraphContent {
+	node := p.expectKeywordLike("NODE").Pos
+	p.expectKeywordLike("TABLES")
+
+	nodeTables := &ast.PropertyGraphNodeTables{
+		Node:   node,
+		Tables: p.parsePropertyGraphElementList(),
+	}
+
+	var edgeTables *ast.PropertyGraphEdgeTables
+	if p.Token.IsKeywordLike("EDGE") {
+		edge := p.expectKeywordLike("EDGE").Pos
+		p.expectKeywordLike("TABLES")
+		tables := p.parsePropertyGraphElementList()
+		edgeTables = &ast.PropertyGraphEdgeTables{
+			Edge:   edge,
+			Tables: tables,
+		}
+	}
+
+	return &ast.PropertyGraphContent{
+		NodeTables: nodeTables,
+		EdgeTables: edgeTables,
+	}
+}
+
+func (p *Parser) parsePropertyGraphElementList() *ast.PropertyGraphElementList {
+	lparen := p.expect("(").Pos
+	elements := parseCommaSeparatedList(p, p.parsePropertyGraphElement)
+	rparen := p.expect(")").Pos
+
+	return &ast.PropertyGraphElementList{
+		Lparen:   lparen,
+		Rparen:   rparen,
+		Elements: elements,
+	}
+
+}
+func (p *Parser) parsePropertyGraphElement() *ast.PropertyGraphElement {
```

Comment:
Insert an empty line.

```suggestion
}

func (p *Parser) parsePropertyGraphElement() *ast.PropertyGraphElement {
```